src/User/User.ts

```diff
@@ -0,0 +1,32 @@
+import firebase from "firebase";
+
+type getToken =
+  | ((forceRefresh?: boolean | undefined) => Promise<string>)
+  | undefined;
```

Comment:
Can we do without `undefined`?

Reply:
https://github.com/pesto-students/juntos-frontend/pull/18#discussion_r649158858